guides/common/modules/con_monitoring_hosts_using-red-hat-insights.adoc

```diff
@@ -0,0 +1,22 @@
+[id="monitoring_hosts_using_red_hat_insights_{context}"]
+= Monitoring Hosts using Red{nbsp}Hat Insights
+
+You can use Insights to diagnose systems and downtime related to security exploits, performance degradation and stability failures.
+You can use the dashboard to quickly identify key risks to stability, security, and performance.
+You can sort by category, view details of the impact and resolution, and then determine what systems are affected.
+
+To use Insights to monitor hosts that you manage with {Project}, you must first install Insights on your hosts and register your hosts with Insights.
+
+For new {Project} hosts, you can install and configure {Project} hosts with Insights during registration using the global registration template.
+For more information, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in _{ManagingHostsDocTitle}_.
+
+To install and register your host using Puppet, or manually, see https://access.redhat.com/products/red-hat-insights/#getstarted[Red{nbsp}Hat Insights Getting Started].
+
+.Additional Information
+* To view the logs for Insights and all plug-ins, go to `/var/log/foreman/production.log`.
+* If you have problems connecting to Insights, ensure that your certificates are up-to-date.
+Refresh your subscription manifest to update your certificates.
+* You can change the default schedule for running `insights-client` by configuring `insights-client.timer` on a host.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_insights/2020-10/html/client_configuration_guide_for_red_hat_insights/changing-the-client-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
+endif::[]
```

Comment:
This content should replace the current content in the existing module that is at the beginning of the assembly.